src/components/button/button.tsx

```diff
@@ -94,6 +94,8 @@ export class Button {
                         outlined: this.outlined,
                     }}
                     disabled={this.disabled || this.loading}
+                    aria-busy={this.loading ? 'true' : 'false'}
+                    aria-live="polite"
```

Comment:
It seems like this might be the wrong commit? This is only changing `limel-button`, not `limel-chip`, and this same  change seems to also be present in #3193?

Reply:
I think you accidentally made a duplicated branch from your other branch where you were working on `limel-button`.

Please make a new branch for chip 😄 and re-link it to the issue.